leanplum_api.gemspec

```diff
@@ -5,24 +5,26 @@ require 'leanplum_api/version'
 Gem::Specification.new do |gem|
   gem.name        = 'leanplum_api'
   gem.version     = LeanplumApi::VERSION
-  gem.authors     = ['Lumos Labs, Inc.']
-  gem.email       = ['analytics-dev@lumoslabs.com']
+  gem.authors     = ['Lumos Labs, Inc.', 'Countable Corp']
+  gem.email       = ['analytics-dev@lumoslabs.com', 'eng@countable.us']
   gem.homepage    = 'http://www.github.com/lumoslabs/leanplum_api'
   gem.summary     = 'Gem for the Leanplum API'
   gem.description = 'Ruby-esque access to Leanplum API'
   gem.licenses    = ['MIT']
   gem.files       = Dir["lib/**/*"] + ['Gemfile', 'LICENSE.txt', 'README.md']
   gem.test_files  = Dir["spec/**/*"]
 
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 2.3'
```

Comment:
agree we should bump this but why does it need to be 2.3?  what feature does 2.3 have that makes it required?

Reply:
speed.

Reply:
that's not a good answer for people still running apps on 2.x - this is overly restrictive for them.

fine to move it to `>= 2.0` but there's no point in restricting the ruby version arbitrarily beyond that unless you _require_ a given feature introduced in 2.x